### DISCUSSION TITLE
NAV/COM Questions

### DISCUSSION THREAD

**Discussion by Suicidal-Gnome** [nguyenquyhy/Flight-Tracker-StreamDeck#360] [Q&A]
Hey Nguyen! Firstly, thanks so much for developing this plugin, especially as freeware! It's fantastic! I have a question I was hoping you could answer:

I am trying to make INC/DEC WHOLE/FRAC buttons repeat on hold, so that I don't have to press each button multiple times to go, for example, from 119.300 to 126.850. I checked the "Repeat" box which worked when I held the button, except that when I would single press the button it would repeat a few times as well and wouldn't singularly INC/DEC. What suggestions might you have?

**Comment by rmroc451** [comment]
Spamming an event tends to cause simconnect issues. Alot of the buttons that have press and hold functionality detect if it has been pressed for a bit and batch commands in higher increment levels.

I think this is still the case. I've seen that a few times while plunking around making updates :)  I'd imagine it's still the case.
